feature-libs/order/components/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -13,6 +13,14 @@
     </ng-container>
 
     <!-- pickup consigned entries -->
+
+    <div
+      *ngIf="pickupConsignments?.length > 0"
```

Comment:
no, either `pickupConsignments?.length` or `pickupConsignments && pickupConsignments.length > 0`.

Reply:
I have made the change. Thank you